src/MEDS_DEV/helpers/extract_task.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export MEDS_ROOT_DIR=$1
+export MEDS_DATASET_NAME=$2
+export MEDS_TASK_NAME=$3
+
+shift 3
+
+MEDS_DEV_REPO_DIR=$(python -c "from importlib.resources import files; print(files(\"MEDS_DEV\"))")
```

Comment:
I'm not sure how I can get this command to run independently for some reason. What is it supposed to return? 😕 

(I get `ModuleNotFoundError: No module named 'MEDS_DEV'`, but on local `pip install .` it does return `Successfully installed MEDS_DEV-0.1.dev38+gb28726e.d20240902`...)

Reply:
hmm -- so you are pip installing . on this branch and this command still doesn't work?

Reply:
Oh I found the mistake—I locally changed `MEDS-DEV` → `MEDS_DEV` in `__init.py__` in my branch but not the folder name :) Thanks!